INSTALL.md

```diff
@@ -106,12 +84,35 @@ If you have an error with this command, try deleting the folder `tools/gulp/node
 
 ## Connection to application
 
+A Django development server is automatically started unless you have run with production settings. It is accessible at `http://localhost:8000`.
+
+## Backup and restore database
+
+The database content is stored in a Docker named volume that is not directly accessible.
+
+**For backup**, running:
+
+```
+$> docker run --rm --volumes-from souschef_db_1 -v $(pwd):/backup ubuntu tar cvf /backup/backup.tar /var/lib/mysql
+```
+
+In Windows console, running:
 
-You should now be ready to run the Django application by pointing your browser to the container address.
+```
+$> docker run --rm --volumes-from souschef_db_1 -v %cd%:/backup ubuntu tar cvf /backup/backup.tar /var/lib/mysql
+```
+
+`souschef_db_1` is the container's name that can be found by running `docker ps`. This command creates a temporary Ubuntu container, connects it with both the volume that `souschef_db_1` uses and current directory on host machine. You will find `backup.tar` in current directory after this command.
+
+**For restoring**:
+
+```
+$> docker run --rm --volumes-from souschef_db_1 -v $(pwd):/backup ubuntu bash -c "cd /var/lib/mysql && tar xvf /backup/backup.tar --strip 1"
+```
 
-On Linux and OS X, open http://localhost:8000.
+In Windows console, replace `$(pwd)` as `%cd%`.
 
```

Comment:
@lingxiaoyang I suggest adding the command line for Windows console, event if the user only needs to replace `$(pwd)` as `%cd%`, you have already put the two versions in the backup section, so it will be nice to remain to document both versions in the restoring section. What do you think?

`$> docker run --rm --volumes-from souschef_db_1 -v %cd%:/backup ubuntu bash -c "cd /var/lib/mysql && tar xvf /backup/backup.tar --strip 1"`